index.js

```diff
@@ -16,35 +19,60 @@ cacheServicesResponse()
   'serviceVersionCreated'
 ].forEach((eventKey) => {
   mesg.listenEvent({ serviceID: 'marketplace', eventFilter: eventKey })
```

Comment:
Instead of listening individually every event, the app could listen for all event from the marketplace and just try to get the `sid` from the event data. If no `sid`, the event is discarded, if an `sid` is present, the refresh cache

Reply:
explicitly listening is better for me. creating a logic depending on presence of `sid` is not enough to explain if the event was related of a change on a service.